lib/publiccloud/instance.pm

```diff
@@ -47,7 +47,7 @@ C<<$instance->username()>>.
 Use argument C<rc_only> to only check for the return code of the command.
 =cut
 sub run_ssh_command {
-    my $self = shift;
+    my ($self) = @_;
     my %args = testapi::compat_args({cmd => undef}, ['cmd'], @_);
```

Comment:
this is going to break the test.
think what happens when you run `run_ssh_command(cmd=>$my_cmd)`

Reply:
Thank you @b10n1k .  Currently, I am still verifying the changes by executing few test suites. I am not sure how to check the above scenarios. 
I had automated these changes and will wait for the input to proceed.